charts/meteor-pipelines/templates/byon-import-jupyterhub-image.yaml

```diff
@@ -34,6 +37,7 @@ spec:
             echo "  Image description: $(params.desc)"
             echo "  Image URL:         $(params.url)"
             echo
+            echo "  Creator:           $(params.creator)"
```

Comment:
I believe we probably want to add a similar line to the output of updating the IS, https://github.com/thoth-station/helm-charts/blob/fb2022093161bb377b18cb35d99e106200657858/charts/meteor-pipelines/templates/byon-import-jupyterhub-image.yaml#L152-L157

Reply:
Hm.. I don't know.. we're not changing that value in the update step. If we were, it should be in the output, if the value stays the same, I don't see a reason for it. We're doing the same with all the other static annotations.

Reply:
WDYT? Should we include it all?

Reply:
You are right, makes sense, let's leave it out there.